WordPress/src/main/res/values/strings.xml

```diff
@@ -321,13 +321,14 @@
     <string name="out_of_memory">Device out of memory</string>
     <string name="file_not_found">Couldn\'t find the file for upload. Was it deleted or moved?</string>
     <string name="download">Downloading media</string>
-    <string name="delete_post">Delete post</string>
-    <string name="delete_page">Delete page</string>
+    <string name="delete_post">Delete post?</string>
+    <string name="delete_page">Delete page?</string>
     <string name="share_url">Share URL</string>
     <string name="posts_fetching">Fetching posts…</string>
     <string name="pages_fetching">Fetching pages…</string>
     <string name="toast_err_post_uploading">Unable to open post while it\'s uploading</string>
     <string name="dialog_confirm_cancel_post_media_uploading">Are you sure? Deleting this post will also cancel the media upload.</string>
+    <string name="dialog_confirm_delete_post_page">This %s will be deleted and sent to Trash</string>
```

Comment:
Please create two strings here:
`This post will be deleted and sent to Trash` and `This page will be deleted and sent to Trash`. "This" might need different translations, depending on the gender of page or post.

Reply:
Thanks for the suggestion, addressed in ec54139